### PR TITLE
[Bug] - Fixed Ordering of List Numbers

### LINKED ISSUE #353: [Bugs] - Wrong Numbering for Lists in Many Sections
**Describe the bug**
The ordering of list items in many sections across the kubeedge docs repo are wrong which need to be corrected. An example for reproducing of the bug is depicted below (Ref: Screenshot below)

**To Reproduce**
1. Go to 'https://kubeedge.io/docs/subproject/ianvs#architecture'
2. Scroll down after the image and you will see the error

**Expected behavior**
The ordering of lists and items should be corrected.

**Screenshots**

Screenshot 2023-07-29 at 1 07 35 PM

### PR DESCRIPTION
**Pull request by krishnasvp**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- Fixes ordering of list numbers across the repository - Fixes #353 

**What is the current behavior?** (You can also link to an open issue here)
- List numbers are repeated, More info #353 

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
- Nope, this PR does not introduce a breaking change

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/lgtm

File: docs/developer/device_crd.md [hunk 1]
```diff
@@ -205,10 +205,10 @@ Note: The creation of a device instance will also lead to the creation of a conf
 
 1. Run the mapper application corresponding to your protocol.
 
-2. Edit the status section of the device instance yaml created in step 2 and apply the yaml to change the state of device twin. This change will be reflected at the edge, through the device controller
+2. Edit the status section of the device instance yaml created in step 3. and apply the yaml to change the state of device twin. This change will be reflected at the edge, through the device controller
  and device twin modules. Based on the updated value of device twin at the edge the mapper will be able to perform its operation on the device.
 
-1. The reported values of the device twin are updated by the mapper application at the edge and this data is synced back to the cloud by the device controller. Users can view the update on the cloud by checking their device instance object.
+4. The reported values of the device twin are updated by the mapper application at the edge and this data is synced back to the cloud by the device controller. Users can view the update on the cloud by checking their device instance object.
```
Comment (kevin-wangzefeng) [inline docs/developer/device_crd.md:211]:
```suggestion
3. The reported values of the device twin are updated by the mapper application at the edge and this data is synced back to the cloud by the device controller. Users can view the update on the cloud by checking their device instance object.
```

**Comment by kevin-wangzefeng** [comment]
/lgtm
/approve
Thanks

**Pull request merged**